### DISCUSSION TITLE
ERR_UNSUPPORTED_API_LEVEL: Unsupported API level: 34

### DISCUSSION THREAD

**Discussion by emmanuel2-s** [ionic-team/capacitor#6914] [Q&A]
Trying to run the android application but am getting this error
undefined
        ERR_UNSUPPORTED_API_LEVEL: Unsupported API level: 34

**Comment by jcesarmobile** [comment]
API 34 is still on beta, so it's not supported by native-run, you'll have to downgrade to 33.
Not sure why google by default downloads SDK 34 when it's still on beta.

**Comment by emmanuel2-s** [comment]
Thanks, I have done it, and it worked.
